Formula/trunk.rb

```diff
@@ -0,0 +1,19 @@
+class Trunk < Formula
+  desc "Build, bundle & ship your Rust WASM application to the web"
+  homepage "https://github.com/thedodd/trunk"
+  url "https://github.com/thedodd/trunk/archive/v0.6.0.tar.gz"
+  sha256 "5492729a6dd027c46d91bacca9b3b87beb3b534c7c6f0f2529f9675600a9e273"
+  license any_of: ["MIT", "Apache-2.0"]
+  head "https://github.com/thedodd/trunk.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    config_first_line = "ConfigOpts {\n"
+    assert_equal config_first_line, shell_output("#{bin}/trunk config show").lines.first
```

Comment:
```suggestion
    match "ConfigOpts {\n", shell_output("#{bin}/trunk config show")
```

Reply:
@SMillerDev that appears to be invalid syntax. Did you mean something like `assert /ConfigOpts {\n/.match?(shell_output("#{bin}/trunk config show"))`

I tried your suggested code, but Ruby does not accept it as valid syntax `:)`.

Also, just so I understand your reasoning, why do you not like what is currently here? Both accomplish the same thing? Some might say that the code which is currently in place is more readable. The "assert_equal x, y" as it currently is seems pretty clear.

**To be sure,** I'm open to whatever as long as it works and accomplishes our goal here. Just let me know.

Reply:
Something went wrong in writing that, edited it and it should work now.

Reply:
Done!